lib/src/iterable_extensions.dart

```diff
@@ -59,7 +70,21 @@ extension NestedIterableExtensions<E> on Iterable<Iterable<E>> {
   /// elements in iteration order.
   ///
   /// `since 0.0.1`
-  Iterable<E?> get flatten {
+  Iterable<E> get flatten {
     return expand(identity);
   }
 }
+
+/// Extends [Iterable] of [Future]s
+///
+/// `since 0.8.0`
+extension FutureIterableExtensions<V> on Iterable<Future<V>> {
+  /// Does a [Future.wait] for all values in this.
```

Comment:
```suggestion
  /// Does a [Future.wait] for all values in [this].
```

Reply:
optional. You did it on the other PR

Reply:
Not `?

Reply:
`[this]` is know in dart doc as the current instance